### DISCUSSION TITLE
How to get Instance tags and more details to Alert messages

### DISCUSSION THREAD

**Discussion by yallaturisatish** [grafana/grafana#46884] [Alerting]
Requirement:
Generally users can easily understand if we show more details about an issue in the notification messages like Instance tags Instead of showing just Instance ID's.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
